cmd/utils/flags.go

```diff
@@ -790,59 +790,31 @@ func MustMakeChainConfigFromDb(ctx *cli.Context, db ethdb.Database) *core.ChainC
 	c := new(core.ChainConfig)
 
 	genesis := core.GetBlock(db, core.GetCanonicalHash(db, 0))
-	if genesis != nil {
-		storedConfig, err := core.GetChainConfig(db, genesis.Hash())
```

Comment:
I guess it was made this way to keep "--oppose-hard-fork" selection. But now, with new forks every week, it doesn't make much sense. 

Better to introduce "--chain" cli option, and store different chains in different directories. Similar to Parity's way